src/app/onion/learning-object-builder/pages/materials-page/materials-page.component.html

```diff
@@ -1,6 +1,6 @@
 <clark-column-wrapper [columns]="'lm'">
-  <div left>
-    <clark-scaffold></clark-scaffold>
+  <div left *ngIf="learningObject">
```

Comment:
Can we move this ngIf to the child element? That way it will at least stop the parent element from snapping in on slow connections

Reply:
Resolved with 3afea18.